package.json

```diff
@@ -31,6 +32,9 @@
   "author": "GitHub Inc.",
   "license": "ISC",
   "dependencies": {
+    "@actions/core": "^1.9.1",
+    "@commitlint/cli": "^17.1.1",
+    "@commitlint/config-conventional": "^17.1.0",
```

Comment:
oooo i see, so we're just letting template-oss specify these and running the bins?

Reply:
Yeah, and using `--offline` ensures we always have it there. The one tradeoff is we need to run `npm install` first.

Reply:
out of scope for this, but i wonder if it makes sense for us to make a proper bin for template-oss that can run these external bins for us with something like `template-oss commitlint`